### PR TITLE
Feat single video page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created Single Video Page Containing:
Video Player
Video Description
Action buttons to like the video, add to history, add to watch later

### CODE REVIEW

**Review by vaibhav-mahalle — APPROVED** [review]
I liked three things from the pull request 

1. Consistent naming conventions followed throughout
2. well-structured code structure
3. well-categorized commit messages

Things you could improve in further pull requests

1. Keep commit messages short and naming could be better.

File: src/Components/index.js [hunk 1]
```diff
@@ -3,11 +3,13 @@ import Menu from "./Menu";
...
 
 export {
     Header,
     Menu,
     FilterPills,
     SideNav,
-    VideoCard
+    VideoCard,
+    VideoActions
 }
```
Comment (rahulcs754) [inline src/Components/index.js:15]: no need this

File: src/Components/index.js [hunk 2]
```diff
@@ -3,11 +3,13 @@ import Menu from "./Menu";
 import FilterPills from "./FilterPills";
 import SideNav from "./SideNav";
 import VideoCard from "./VideoCard";
+import VideoActions from "./VideoActions";
```
Comment (rahulcs754) [inline src/Components/index.js:6]: you can write directly export not import so you have to not write again export

**Review by rahulcs754 — APPROVED** [review]
The code look clean and good 
just small suggestion

**Pull request merged**